vector/src/main/res/layout/dialog_invite_by_id.xml

```diff
@@ -0,0 +1,37 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/bug_report_body_view"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="10dp"
+    android:layout_marginLeft="10dp"
+    android:layout_marginRight="10dp"
+    android:layout_marginTop="10dp"
```

Comment:
Could use `android:layout_margin="10dp"` instead.  
See https://material.io/guidelines/components/dialogs.html#dialogs-specs for recommended margins.